### DISCUSSION TITLE
Motion detection start triggers from a superview angle, even when video set in a linear

### DISCUSSION THREAD

**Discussion by shaaimars** [gopro/labs#1018] [Q&A]
Today i updated my gopro and "qr control" app, and started testing motion sense feature. For a while i was pissed off, because it felt like a complete madness - it was turning on and off without nothing going on on a gopro screen. After little bit of testing this is what i found:

1) Motion sence VIDEO START trigger working in an "superview angle", so you could see no movement on a gopro screen, but the camera would start recording. MISC jpg file is also captured in a superview frame.
2) As soon as it starts recording it works just fine, according to the lense mode camera set in, it can be seen by a "X difference" caption. So the "pissing off" behavior is:  camera sees movement inside superview and outside linear frame, starts recording and then immideatly stops seeing the movement, stops recording, and repeat. 

I think i've provided enough instructions, but let me know if you need a video/photo of the bug. 
The qr code is this (https://github.com/user-attachments/assets/9988eb8b-5827-4212-a2fb-ab1802f052ce)

**Comment by dnewman-gpsw** [comment]
You have also mention the key information, what camera, firmware version, video mode, frame rate, etc.

**Comment by shaaimars** [comment]
Yes, i forgot it. 
Camera is HERO 11 black, firmwave version is Labs v2.30.70
Video mode is 4k 30fps, linear, zoom 100%
Motion detection start sensitivity 5, stop sensitivity 5.

**Comment by dnewman-gpsw** [comment]
Your findings are correct. As I do I my testing in wide I hadn't noticed the discrepancy.   
This might not be easily fixable, as the camera is not using LCD buffer for detection, rather is closer to the raw sensor data.  It did use the LCD buffer on HERO10, but that required to the screen to be on (user did not like that.)  When the capture starts the buffer used is formatted image using for encoding, that is the correct FOV, causing the mis-match.  There is a partial crude workaround using a mask.pgm file to indicate which pixel are used for motion detection, but that will be correct for the start, but wrong for the end of capture.
